Makefile

```diff
@@ -2,6 +2,10 @@ opnode:
 	go build -o ./bin/op ./opnode/cmd
 .PHONY: opnode
 
+contracts:
+	cd packages/contracts && yarn build
```

Comment:
this doesn't seem to work on macos. 
This works:
```suggestion
	cd ./packages/contracts && yarn build
```

Reply:
Should it also do `yarn install` before build in case of a fresh clone?

Reply:
I think the error message when that happens is pretty clear.